notification-service/src/generators/generateDiscordMessage.ts

```diff
@@ -0,0 +1,44 @@
+import { EventSubscription } from '../types';
+import { formatEtherscanLink } from '../etherscan';
+
+export function generateDiscordMessage(
+    network: string,
+    eventSubscription: EventSubscription,
+    event: any,
+    formattedData: string
+) {
+    return {
+        embeds: [
+            {
+                author: {
+                    name: 'Unified Auctions Service',
+                    url: 'https://github.com/sidestream-tech/unified-auctions-ui',
```

Comment:
I would also never hardcode those kind of values, but at least provide them as default via env variables

Reply:
Updated it!